### ISSUE TITLE
nbopen doesn't create file it it doesn't exist

### ISSUE THREAD

**Issue by ctb** [takluyver/nbopen#65]
e.g.
`nbopen foo.ipynb` opens the JupyterLab interface if `foo.ipynb` doesn't exist.

Ideally, it would create an empty file of that name. I bet the tricky bit is that you have to guess at the format of the file based on the name...

**Comment by ctb** [comment]
Whups! Realized `-n/--new` does this just fine! I feel like this could/should be the default tho?

**Comment by takluyver** [comment]
Yup, it should either create the file, or fail consistently with a short, clear error message. At present, it fails with a load of Jupyter messages ending in a reasonable error if there isn't a server running, and opens a 404 page if there is.

I'll have a think about which I prefer.

**Comment by ctb** [comment]
I like creating the file, personally. I will see if I can hack on this and
submit a PR!

**Comment by takluyver** [comment]
It should be pretty straightforward :-). If you can get a PR in before I've decided what I prefer, I'll go with what you want.
